InvenTree/part/templates/part/build.html

```diff
@@ -1,15 +1,18 @@
 {% extends "part/part_base.html" %}
 {% load static %}
+{% load i18n %}
 {% block details %}
 
 {% include 'part/tabs.html' with tab='build' %}
 
-<h3>Part Builds</h3>
+<h3>{% trans "Part Builds" %}</h3>
 
 <div id='button-toolbar'>
     <div class='button-toolbar container-flui' style='float: right';>
         {% if part.active %}
-        <button class="btn btn-success" id='start-build'>Start New Build</button>
+        {% if roles.build.add %}
```

Comment:
@eeintech checking roles in templates is now done this way as per your suggestion

Reply:
Is the `user_roles` evaluated on each request? Or how is the context updated when user makes a ruleset change?